inst/eDISH_app/modules/dataUpload/dataUpload.R

```diff
@@ -1,7 +1,10 @@
 dataUpload <- function(input, output, session){
   
-  ns <- session$ns
+  maxFileSize <- 20
```

Comment:
I was thinking we'd handle this in `safetyGraphicsApp()`. That we could make `maxFileSize` a new option that the user can configure for a given session.